CHANGES.rst

```diff
@@ -1,6 +1,23 @@
 Changes
 =======
 
+Version 0.6.0 (2019-12-20)
+--------------------------
+
+- Adds new flag and configuration to optionally deploy with CERN storage EOS.
+- Creates a new Kubernetes service account for REANA and associates the
+  persmissions REANA needs.
+- Moves Traefik installation to Helm 3.0.0.
```

Comment:
Shall we mention above Helm and Kubernetes upgrades?

Reply:
I think it makes sense, it's a big change.

Reply:
I will for Kubernetes thanks. Regarding Helm, it is just for Traefik so this line should be enough.